src/ontology/uberon-edit.obo

```diff
@@ -222484,6 +222484,17 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500106
+name: hormion
+def: "Craniometric point along the median plane at the ventral (inferior in humans) end of the suture of the vomer and sphenoid bones." [ISBN:9780873659505]
+synonym: "vomerine notch" EXACT []
+is_a: UBERON:0004120 ! mesoderm-derived structure
+is_a: UBERON:0005913 ! zone of bone organ
+relationship: part_of UBERON:0002396 ! vomer
+property_value: http://purl.org/dc/elements/1.1/contributor http://orcid.org/0000-0003-2699-3066
```

Comment:
should be dc terms, not dc elements :)

Reply:
protege > preferences > new entities metadata > creator property> "http://purl.org/dc/terms/contributor" :) lmk if any of that is unclear